xml/System.Text.Json/JsonWriterOptions.xml

```diff
@@ -55,9 +58,19 @@
         <ReturnType>System.Boolean</ReturnType>
       </ReturnValue>
       <Docs>
-        <summary>To be added.</summary>
-        <value>To be added.</value>
-        <remarks>To be added.</remarks>
+        <summary>Gets or sets a value that indicates whether the <see cref="T:System.Text.Json.Utf8JsonWriter" /> should skip structural validation and allow the user to write invalid JSON.</summary>
+        <value><see langword="true" /> to skip structural validation and allow invalid JSON; otherwise, <see langword="false" />: any attempts to write invalid JSON will result in
```

Comment:
Is using `:` correct here? Can you share the link to the docs so I can see how it would look?

Reply:
I'm not totally happy with this description. I'll see if I can improve this a bit.

https://review.docs.microsoft.com/en-us/dotnet/api/system.text.json.jsonwriteroptions.skipvalidation?view=netcore-3.0&branch=pr-en-us-1880